plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -144,9 +146,11 @@ func (t *TokenAuthenticator) AuthenticateToken(token string) (user.Info, bool, e
 		return nil, false, nil
 	}
 
-	return &user.DefaultInfo{
-		Name:   bootstrapapi.BootstrapUserPrefix + string(id),
-		Groups: groups,
+	return &authenticator.Response{
```

Comment:
TODO to take apiserver audiences at `TokenAuthenticator ` construction time, and if ctx is audience-limited, ensure it intersects with the apiserver audiences, and echo an audience in the response